src/containers/form/inputs/AgeInput.js

```diff
@@ -13,7 +13,7 @@ class AgeInput extends Component {
         this.setWaitingStatus = this.setWaitingStatus.bind(this);
     }
     validation() {
-        this.props.validate(this.regExp.test(this.props.age.value));
+        this.props.validate(this.regExp.test(this.props.age.value), I18n.t('application.form.errors.ageError'));
```

Comment:
Why don't you use I18n.t(...) everywhere? Somewhere you use Translate components and here I see I18n.t... What is the purpose of using two different ways of internationalization?